src/classes/RD2_ScheduleService.cls

```diff
@@ -277,8 +277,7 @@ public without sharing class RD2_ScheduleService {
 
         Boolean updateNeeded = false;
 
-        if (newRd.Status__c == RD2_Constants.STATUS_LAPSED ||
-                (oldRd.Status__c == RD2_Constants.STATUS_LAPSED && newRd.Status__c == RD2_Constants.STATUS_ACTIVE)) {
+        if (oldRd.Status__c == RD2_Constants.STATUS_LAPSED && newRd.Status__c == RD2_Constants.STATUS_ACTIVE) {
```

Comment:
@npsp-reedestockton  What happens if the status was Active is is changed to Lapsed?

Reply:
Yep.  That is an issue.  

I'm going to take the time now to rework this qualification logic.  It is getting way too spread out.  It is starting to remind me of the Ptolemaic explanations for planetary orbits.  Something wrong?  Add a new epicycle...

We need a single source of truth and one place to maintain the logic.